kontrakter/dom/dom/arbeidsversjon/eksempelfiler/eksempel_dom.json

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
Jostein, Anne Lise og jeg har undersøkt litt rundt hvilke reaksjoner som er viktigst for oss i KO i sakene vi ser på nå.

Vi ønsker selvfølgelig mest mulig strukturert, men disse reaksjonstypene er de vi har mest av i systemene vår p.t.:
* ubetinget fengsel
* samfunnsstraff
* deldom
* bet. dom med RK-program